### PR TITLE
[Doctest] Add configuration_deberta.py

### PR DESCRIPTION
**Pull request by Saad135**
# What does this PR do?

Adds configuration_deberta.py to utils/documentation_tests.txt

Based on https://github.com/huggingface/transformers/issues/19487

@ydshieh can you please have a look? thanks :D

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: src/transformers/models/deberta/configuration_deberta.py [hunk 1]
```diff
@@ -90,7 +90,21 @@ class DebertaConfig(PretrainedConfig):
...
         layer_norm_eps (`float`, optional, defaults to 1e-12):
             The epsilon used by the layer normalization layers.
-    """
+
+    Example:
+
+    ```python
+    >>> from transformers import DebertaConfig, DebertaModel
+
+    >>> # Initializing a DeBERTa style configuration
```
Comment (ydshieh) [inline src/transformers/models/deberta/configuration_deberta.py:99]: Let's make it a bit more precise :-)

```python
>>> # Initializing a DeBERTa microsoft/deberta-base style configuration
```

You can check what has been done for `BertConfig`.

File: src/transformers/models/deberta/configuration_deberta.py [hunk 2]
```diff
@@ -90,7 +90,21 @@ class DebertaConfig(PretrainedConfig):
...
+    >>> configuration = DebertaConfig()
+
+    >>> # Initializing a model (with random weights) from the DeBERTa style configuration
```
Comment (ydshieh) [inline src/transformers/models/deberta/configuration_deberta.py:102]: Same as above

**Review by ydshieh — APPROVED** [review]
Thanks a lot, @Saad135! Just 2 suggestions for tiny improvements.

File: src/transformers/models/deberta/configuration_deberta.py (hunk 1 above)
Comment (Saad135) [reply src/transformers/models/deberta/configuration_deberta.py]: Done :D

File: src/transformers/models/deberta/configuration_deberta.py (hunk 2 above)
Comment (Saad135) [reply src/transformers/models/deberta/configuration_deberta.py]: Done :D

File: src/transformers/models/deberta/configuration_deberta.py [hunk 3]
```diff
@@ -90,7 +90,21 @@ class DebertaConfig(PretrainedConfig):
...
+
+    Example:
+
+    ```python
+    >>> from transformers import DebertaConfig, DebertaModel
+
+    >>> # Initializing a DeBERTa microsoft/deberta-base style configuration
+    >>> configuration = DebertaConfig()
+
+    >>> # Initializing a model (with random weights) from the DeBERTa microsoft/deberta-base style configuration
```
Comment (ydshieh) [inline src/transformers/models/deberta/configuration_deberta.py:102]:
```suggestion
    >>> # Initializing a model (with random weights) from the microsoft/deberta-base style configuration
```

Comment (ydshieh) [reply src/transformers/models/deberta/configuration_deberta.py:102]: Just a really tiny thing :-) 🙏 and we can merge

**Review by ydshieh — CHANGES_REQUESTED** [review]

File: src/transformers/models/deberta/configuration_deberta.py (hunk 3 above)
Comment (Saad135) [reply src/transformers/models/deberta/configuration_deberta.py]: Done :-)

**Review by ydshieh — APPROVED** [review]
Thank you again @Saad135 💯

**Pull request merged**